### DISCUSSION TITLE
Login-Frage nach Erstinstallation

### DISCUSSION THREAD

**Discussion by Lacky16H** [camueller/SmartApplianceEnabler#173] [Installation und Betrieb]
Hallo Axel,

dann starte ich mal hier.

Habe gerade versucht die automatische Installation durchlaufen zu lassen. Habe auch keine Fehlermeldung o.ä. erhalten.

der Raspi startet aber nicht (so wie ich es erwartet hätte) mit einer GUI, sondern erwartet noch ein Login von mir.

"raspberrypi login:"

Kannst du mir sagen, was ich da noch eingeben muss?

Habe mal "sae" "sae" probiert, das ging aber nicht.

Mario

**Comment by Lacky16H** [comment]
>
![IMG_7864](https://user-images.githubusercontent.com/98414162/151022238-ecf7d055-c475-43c9-9fee-75b81bc05d3e.jpeg)

**Comment by camueller** [comment]
Dann hat wohl die Installation nicht geklappt, ansonsten sollte es mit Username "sae" und Passwort "sae" funktionieren. Vermutlich hast Du Dich irgendwo nicht genau an die Anleitung gehalten.
Einloggen kannst Du Dich aber auch mit Username "pi" und Passwort "raspberry", aber das wird Dir wohl nicht viel helfen bzgl. des SAE.
Eine grafische Oberfläche verwendet das Image nicht, dass ich für die automatische Installation verwende.

**Comment by camueller** [comment]
Ich bin gerade dabei, die automatische Installation automatisiert zu testen. In der Vergangenheit kam es schon einmal vor, dass die Installation tatsächlich nicht mehr funktioniert hat. Deshalb habe ich diese Automatisierung schon länger vor, wobei ich noch nicht sicher bin, ob die Automatisierung so funktionieren wird. Danach werden wir hoffentlich wissen, ob die automatische Installation aktuell immer noch funktioniert.

**Reply by Lacky16H** [reply]
danke, das ist prima

nachdem mein heutiger Versuch an der gleichen Stelle endete wie mein gestriger habe ich meinen Raspi mittlerweile auf Ebay-Kleinanzeigen eingestellt und mein Projekt für beendet erklärt.

Dann werde ich die Anzeige noch mal pausieren stellen und hoffen ;-)

**Reply by camueller** [reply]
Bei meinem Test gestern ist die Installation fehlerfrei durchgelaufen. Es liegt also nicht an der Installation oder an den während der Installation nachgeladenen Packages. Die Erfahrung mit der automatischen Installation hat gezeigt, dass es meist daran lag, dass bei Windows-Usern an die Dateien irgendwelche Erweiterungen angehangen wurden. Hast Du das neueste Image von https://github.com/camueller/RaspiOSImageAutorunUSBShellScript/releases verwendet?

**Reply by camueller** [reply]
Ich bin immer noch dabei, das automatisierte Testen der automatischen Installation zu "bauen". Dabei hat sich jetzt herausgestellt, dass der SAE danach nicht gestartet werden kann, weil Java nicht installiert werden konnte. Ich muss noch untersuchen, ob das am Test in einem virutellen Raspberry Pi liegt, oder ob es auch auf einem realen System auftritt.

**Reply by srype** [reply]
Kämpfe momentan mit dem selben Problem. Der Standard Raspy Login funktioniert. Die automatische Installation scheint aber nicht zu funktionieren.

**Reply by camueller** [reply]
Laut [diesem Beitrag im PV-Forum](https://www.photovoltaikforum.com/thread/104060-ger%C3%A4te-mit-home-manager-koppeln-via-semp-ethernet/?postID=2534442#post2534442) funktioniert die automatische Installation aktuell.

**Reply by srype** [reply]
Aber was mache ich dann falsch? Wenn ich das OS auf dem Raspy starte kommt erst der Login und dann nur noch die Kommandozeile. Wenn ich dann den USB-Stick mit den Dateien einstecke passiert nichts. Was ist da schief gegangen?
Edit: Die Probleme habe ich mit der 2021-05-07 und der 2021-03-04

**Comment by Lacky16H** [comment]
Hi, hilft dir leider nicht. Aber das gleiche Problem habe ich seit Wochen leider auch.

**Reply by srype** [reply]
Naja irgendwer wird doch wohl helfen können bei dem Problem. Ich habe mich zu 100% an die Anleitung gehalten. Der automatische start der Skripte scheint nicht zu funktionieren. Liegt es am falschen Dateiformat der USB-Sticks? @camueller

**Comment by camueller** [comment]
Man kann immer die manuelle Installation durchführen, falls die automatische aus irgendwelchen Gründen nicht geht. Erstere gibt es erst seit gut einem Jahr - vorher musst man manuell installieren. Ist ja auch nicht schwer - einfach die Befehle aus der Doku per Copy&Paste.

**Reply by srype** [reply]
Dann werde ich das bei Gelegenheit probieren. Vielen Dank 🤝

**Comment by camueller** [comment]
Ich habe gerade selbst die Installation getestet. **Dabei konnte ich keinen Fehler feststellen!**

Die Dokumentation habe ich erweitert um Hinweise zur Fehlersuche:
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Installation_DE.md#fehler-bei-der-installation
